compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/PsiUtils.kt

```diff
@@ -461,7 +320,7 @@ public fun PsiElement.requireFqName(
     ?.let { return it }
 
   findFqNameInSuperTypes(module, classReference)
-    ?.let { return it }
+    ?.let { return it.fqName }
```

Comment:
Nit: naming feels a little off here now because it reads as `findFqName().fqName` where we're passing in something called `classReference` which seems like it would have the `FqName`. The following would feel better:

```
findClassReferenceInSuperTypes(module, className)
  ?.let { return it.fqName }
```

Reply:
Good catch. Updated